src/development/form-security.md

```diff
@@ -65,3 +65,18 @@ function joe-plugin-delete-everything() {
     // Now delete everything
     ...
 ```
+
+## Define custom nonce lifetime
+
+By default, nonces are valid 12 hours. From [the source](https://github.com/YOURLS/YOURLS/blob/1.8.2/includes/Config/Config.php#L192-L194):
+```php
+// life span of a nonce in seconds
+if (!defined( 'YOURLS_NONCE_LIFE' ))
+    define( 'YOURLS_NONCE_LIFE', 43200 ); // 3600 * 12
+```
+
+If you want to enforce a different time span, for instance an hour (3600 seconds) you can add the following to your `config.php` :
```

Comment:
```suggestion
If you want to enforce a different time span, for instance an hour (3600 seconds) you can add the following to your `config.php`:

```

Reply:
I'll never get used to this English punctuation rule ... :)